content/blog/2020-04-16-april-20-community-gems.md

```diff
@@ -0,0 +1,151 @@
+
+```dvc
+$ dvc pull process_data_stage.dvc
+```
+
+You can also use `dvc pull` at the level of individual files. This might be
+needed if your DVC pipeline file creates 10 outputs, for example, and you only
+want to pull one (say, `model.pkl`, your trained model) from remote DVC storage.
+You'd simply run
+
+```dvc
+$ dvc pull model.pkl
+```
+
+### Q: [How can I remove a `.dvc` file, but keep the associated files in my workspace?](https://discordapp.com/channels/485586884165107732/485596304961962003/689827778358673469)
+
+Sometimes, you realize you don't want to put a file under DVC tracking after
+all. That's okay, easy to fix. Simply remove the `.dvc` file like any other-
+`rm <file>.dvc`. DVC will then stop tracking the file, and the associated target
+file will still be in your local workspace. Note that the file will still be in
+your
+[DVC cache](https://dvc.org/doc/user-guide/dvc-files-and-directories#structure-of-cache-directory)
+unless you clear it with `dvc gc`.
+
+### Q: [I'm trying to move a stage file with `dvc move`, but I'm getting an error. What's going on?](https://discordapp.com/channels/485586884165107732/563406153334128681/685125650901630996)
+
+The `dvc move` command is used to rename a file or directory and simultaneously
+modify its corresponding DVC file. It's handy so you don't rename a file in your
+local workspace that's under DVC tracking without updating DVC to the change
+(see an [example here](https://dvc.org/doc/command-reference/move#description)).
+The function doesn't work on
+[stage files](https://dvc.org/doc/tutorials/pipelines#define-stages) from DVC
+pipelines. There's not currently an easy way to safely move stage files, and
+it's an
+[open issue we're working on](https://github.com/iterative/dvc/issues/1489).
+Until then, you can manually update the stage file, or make a new one in the
+desired location.
+
+### Q: [I just starting using DVC and noticed that when I `dvc push` files to remote cloud storage, the directory in my remote looks like my DVC cache, not my local workspace directory. Is this right?](https://discordapp.com/channels/485586884165107732/485596304961962003/693740598498426930)
+
+Yep, that's exactly how it should be! In order to provide deduplication and some
+other optimizations, your DVC remote's directory structure will mirror the DVC
+cache (which is by default in your local workspace under `.dvc/cache`).
+Effectively, DVC uses your Git repository to store DVC files, which are keys for
+cache files on your remote. So looking inside your remote won't be particularly
+enlightening if you're looking for human-readable filenames- the file names will
+look like hashes (because, well, they are). Luckily, DVC handles all the
+conversions between the filenames in your local workspace and these hashes. To
+get some more intuition about this, check out some of our
+[docs](https://dvc.org/doc/command-reference/import#examples) about `dvc import`
```

Comment:
can we find a better link that explains how data is stored? import/get is a bit strange choice for that.

Reply:
Agreed!

Reply:
how about this one? I though you were not resolving it for a reason

Reply:
it should be edited. not sure why it isn't showing that code snippet as outdated